host-controller/src/main/resources/org/jboss/as/domain/controller/resources/LocalDescriptions.properties

```diff
@@ -76,6 +76,7 @@ server-group.add.jvm=The jvm name.
 server-group.remove=Remove an existing new server group.
 server-group.profile=The profile name.
 server-group.jvm=The named jvm.
+server-group.graceful-startup=Start the servers in gracefully, queuing incoming requests until the server is fully started.
```

Comment:
s/in gracefully/gracefully

s/queuing incoming/queueing or cleanly rejecting/g